fetch.bs

```diff
@@ -24,7 +24,7 @@ Translate IDs: typedefdef-bodyinit bodyinit,typedefdef-responsebodyinit response
 
 <pre class=anchors>
 url:https://xhr.spec.whatwg.org/#concept-formdata-entry;spec:XHR;type:dfn;text:entries
-url:https://heycam.github.io/webidl/#es-ByteString;type:dfn;text:converted to ByteString;spec:WEBIDL
+url:https://heycam.github.io/webidl/#record-mappings;type:dfn;text:mapping;spec:WEBIDL
```

Comment:
"mappings" should soon be picked up per the change to IDL today. My plan was on updating this PR after that.